### PR TITLE
Using proper RHCOS name

### PR DESCRIPTION
**Pull request by nstielau**
Guidance is:
* Use "Red Hat Enterprise Linux" the first time in readme
* Use RHCOS thereafter
* "RHEL CoreOS" is not a thing

### CODE REVIEW

**Comment by mrguitar** [comment]
Don't forget the CoreOS part: Use "Red Hat Enterprise Linux CoreOS" the first time in readme

**Comment by dustymabe** [comment]
@mrguitar once you LGTM then I'll merge

**Comment by mrguitar** [comment]
lgtm

**Comment by jlebon** [comment]
There's two other references to "RHEL CoreOS" lower down in the doc.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # coreos-installer
 
-`coreos-installer` is a script to install Fedora or Red Hat Enterprise Linux CoreOS to
-target disk. It can be invoked as a standalone script or during bootup
-via a dracut module.
+`coreos-installer` is a script to install Fedora CoreOS (FCOS) or Red Hat 
+Enterprise Linux CoreOS (RHCOS) totarget disk. It can be invoked as a
```
Comment (dustymabe) [inline README.md]: `totarget` - add space

Comment (dustymabe) [reply README.md]: maybe 'to a target disk'

**Pull request merged**